### PR TITLE
nl(contributing): translates code of conduct

### PR DESCRIPTION
**Pull request by joranquinten**

### CODE REVIEW

File: docs/contributing/code-of-conduct.md [hunk 1]
```diff
@@ -1,76 +1,51 @@
...
 
 ## Attribution
 
-This Code of Conduct is adapted from the Contributor Covenant, version 1.4.
+Deze Gedragscode is overgenomen uit het Bijdragerconvenant, versie 1.4.
 
-- homepage: https://contributor-covenant.org
-- version: https://contributor-covenant.org/version/1/4/
+- startpagina: https://contributor-covenant.org
+- versie: https://contributor-covenant.org/version/1/4/
```
Comment (Haroenv) [inline docs/contributing/code-of-conduct.md]: de echte vertaling ziet er anders uit, is dat verwacht?

zie: https://www.contributor-covenant.org/nl/version/1/4/code-of-conduct

Comment (joranquinten) [reply docs/contributing/code-of-conduct.md]: Makes sense to point to the Dutch translation as well. Updated the link. Or did you mean something else?

Comment (MichaelDeBoey) [reply docs/contributing/code-of-conduct.md]: I think we can just copy/past everything from the original, no? 🤔

Comment (Haroenv) [reply docs/contributing/code-of-conduct.md]: I meant that the text of this document isn't the same as the one on the link

Comment (joranquinten) [reply docs/contributing/code-of-conduct.md]: 😅 Ah, sure!

File: docs/contributing/code-of-conduct.md [hunk 2]
```diff
@@ -1,76 +1,48 @@
...
+## Handhaving
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at
-[conduct@gatsbyjs.com](mailto:conduct@gatsbyjs.com). All complaints will be
-reviewed and investigated and will result in a response that is deemed necessary
-and appropriate to the circumstances. The project team is obligated to maintain
-confidentiality with regard to the reporter of an incident. Further details of
-specific enforcement policies may be posted separately.
+Instanties van beledigend, storend of anders niet aanvaardbaar gedrag mag worden gerapporteerd aan het project team op [Conduct@gatsbyjs.com](mailto:conduct@gatsbyjs.com). Alle klachten zullen besproken en onderzocht worden en zullen resulteren in een antwoord dat nodig en passend wordt geacht al naargelang de situatie. Het project team is verplicht om vertrouwelijkheid te garanderen met betrekking tot de klager. Verdere details van specifiek handhavingsbeleid kan apart worden gepost.
```
Comment (MichaelDeBoey) [inline docs/contributing/code-of-conduct.md]:
```suggestion
Instanties van beledigend, storend of anders niet aanvaardbaar gedrag mag worden gerapporteerd aan het project team op [conduct@gatsbyjs.com](mailto:conduct@gatsbyjs.com). Alle klachten zullen besproken en onderzocht worden en zullen resulteren in een antwoord dat nodig en passend wordt geacht al naargelang de situatie. Het project team is verplicht om vertrouwelijkheid te garanderen met betrekking tot de klager. Verdere details van specifiek handhavingsbeleid kan apart worden gepost.
```

File: docs/contributing/code-of-conduct.md [hunk 3]
```diff
@@ -1,76 +1,48 @@
 ---
-title: Gatsby Contributor Covenant Code of Conduct
+title: Gatsby Contributor Covenant Gedragscode
```
Comment (MichaelDeBoey) [inline docs/contributing/code-of-conduct.md:2]:
```suggestion
title: Gatsby Bijdragersconvenant Gedragscode
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Comment by joranquinten** [comment]
@MichaelDeBoey Can you check this one out?

File: docs/contributing/code-of-conduct.md [hunk 4]
```diff
@@ -1,76 +1,48 @@
...
 
 ## Attribution
 
-This Code of Conduct is adapted from the Contributor Covenant, version 1.4.
+Deze Gedragscode is overgenomen uit het Bijdragersconvenant, versie 1.4.
 
-- homepage: https://contributor-covenant.org
-- version: https://contributor-covenant.org/version/1/4/
+- Homepage: https://contributor-covenant.org
+- Versie: https://www.contributor-covenant.org/nl/version/1/4/code-of-conduct
```
Comment (MichaelDeBoey) [inline docs/contributing/code-of-conduct.md]:
```md
## Attributie

Deze Gedragscode is een  aangepaste versie van  de [Contributor Covenant][homepage],
versie 1.4, beschikbaar op https://www.contributor-covenant.org/nl/version/1/4/code-of-conduct.html

[homepage]: https://www.contributor-covenant.org
```

File: docs/contributing/code-of-conduct.md [hunk 5]
```diff
@@ -1,76 +1,48 @@
...
-## Scope
+## Bereik
 
-This Code of Conduct applies both within project spaces and in public spaces
-when an individual is representing the project or its community. Examples of
-representing a project or community include using an official project e-mail
-address, posting via an official social media account, or acting as an appointed
-representative at an online or offline event. Representation of a project may be
-further defined and clarified by project maintainers.
+Deze Gedragscode geldt voor zowel binnen project ruimtes alsook in publieke ruimtes wanneer een individu het project of de gemeenschap vertegenwoordigd. Voorbeelden van vertegenwoordiging van een project of gemeenschap omvatten het gebruik van een officieel email adres van het project, het posten via een officiele sociale media account, of het handelen als een aangestelde vertegenwoordiger op een online of offline event. Vertegenwoordiging van een project mag verder worden gedefinieerd door de projectonderhouders.
```
Comment (MichaelDeBoey) [inline docs/contributing/code-of-conduct.md]:
```suggestion
Deze Gedragscode geldt voor zowel binnen project ruimtes alsook in publieke ruimtes wanneer een individu het project of de gemeenschap vertegenwoordigd. Voorbeelden van vertegenwoordiging van een project of gemeenschap omvatten het gebruik van een officieel email adres van het project, het posten via een officieel sociale media account, of het handelen als een aangestelde vertegenwoordiger op een online of offline event. Vertegenwoordiging van een project mag verder worden gedefinieerd door de projectonderhouders.
```

File: docs/contributing/code-of-conduct.md [hunk 6]
```diff
@@ -1,76 +1,48 @@
...
-## Our Pledge
+## Onze belofte
 
-In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
-our community a harassment-free experience for everyone, regardless of age, body
-size, disability, ethnicity, gender identity and expression, level of
-experience, nationality, personal appearance, race, religion, or sexual identity
-and orientation.
+In het belang van het bevorderen van een open en welkomende omgeving, beloven wij, de bijdragers en onderhouders, om het deelnemen in ons project en onze gemeenschap een intimidatie-vrije ervaring te maken voor iedereen. Dit onafhankelijk van leeftijd, lichaam, handicap, etniciteit, gender identiteit en uiting, ervaringsniveau, nationaliteit, persoonlijk voorkomen, ras, religie of seksuele indentiteit en orientatie.
```
Comment (MichaelDeBoey) [inline docs/contributing/code-of-conduct.md]:
```suggestion
In het belang van het bevorderen van een open en welkomende omgeving, beloven wij, de bijdragers en onderhouders, om het deelnemen in ons project en onze gemeenschap een intimidatie-vrije ervaring te maken voor iedereen. Dit onafhankelijk van leeftijd, lichaam, handicap, etniciteit, gender identiteit en uiting, ervaringsniveau, nationaliteit, persoonlijk voorkomen, ras, religie of seksuele indentiteit en oriëntatie.
```

File: docs/contributing/code-of-conduct.md [hunk 7]
```diff
@@ -1,76 +1,48 @@
...
+- Het publiceren van prive informatie zoals een fysiek of electronisch adres zonder expliciete toestemming
+- Ander gedrag dat redelijkerwijs zou kunnen worden beschouwd als ongepast in een professionele omgeving
 
-## Our Responsibilities
+## Onze verantwoordelijkheden
 
-Project maintainers are responsible for clarifying the standards of acceptable
-behavior and are expected to take appropriate and fair corrective action in
-response to any instances of unacceptable behavior.
+Projectonderhouders zijn verantwoordelijk om klaarheid te scheppen met betrekking tot de standaarden van acceptabel gedrag en worden verwacht de passende en bilijke corrigerende maatregelen te treffen als antwoord op vertoningen van niet acceptabel gedrag.
```
Comment (MichaelDeBoey) [inline docs/contributing/code-of-conduct.md]:
```suggestion
Projectonderhouders zijn verantwoordelijk om klaarheid te scheppen met betrekking tot de standaarden van acceptabel gedrag en worden verwacht de passende en billijke corrigerende maatregelen te treffen als antwoord op vertoningen van niet acceptabel gedrag.
```

File: docs/contributing/code-of-conduct.md [hunk 8]
```diff
@@ -1,76 +1,48 @@
...
 
-Project maintainers have the right and responsibility to remove, edit, or reject
-comments, commits, code, wiki edits, issues, and other contributions that are
-not aligned with this code of conduct, or to ban temporarily or permanently any
-contributor for other behaviors that they deem inappropriate, threatening,
-offensive, or harmful.
+Projectonderhouders hebben het recht en de verantwoordelijkheid om commentaar, commits, code, wiki edits, issues en andere contributies te verwijderen, te bewerken of af te keuren indien deze niet voldoen aan deze Gedragscode, of om tijdelijk of permantent te bannen zij wiens gedrag als ongewenst, bedreigend, beledigend of schadelijk wordt geacht.
```
Comment (MichaelDeBoey) [inline docs/contributing/code-of-conduct.md]:
```suggestion
Projectonderhouders hebben het recht en de verantwoordelijkheid om commentaar, commits, code, wiki edits, issues en andere contributies te verwijderen, te bewerken of af te keuren indien deze niet voldoen aan deze Gedragscode, of om tijdelijk of permanent te bannen zij wiens gedrag als ongewenst, bedreigend, beledigend of schadelijk wordt geacht.
```

File: docs/contributing/code-of-conduct.md (hunk 5 above)
Comment (MichaelDeBoey) [inline docs/contributing/code-of-conduct.md]:
```suggestion
Deze Gedragscode geldt voor zowel binnen project ruimtes alsook in publieke ruimtes wanneer een individu het project of de gemeenschap vertegenwoordigd. Voorbeelden van vertegenwoordiging van een project of gemeenschap omvatten het gebruik van een officieel email adres van het project, het posten via een officieel sociale media account, of het handelen als een aangestelde vertegenwoordiger op een online of offline event. Vertegenwoordiging van een project mag verder worden gedefinieerd door de projectonderhouders.
```

File: docs/contributing/code-of-conduct.md (hunk 4 above)
Comment (MichaelDeBoey) [inline docs/contributing/code-of-conduct.md]:
```md
## Attributie

Deze Gedragscode is een  aangepaste versie van  de [Contributor Covenant][homepage],
versie 1.4, beschikbaar op https://www.contributor-covenant.org/nl/version/1/4/code-of-conduct.html

[homepage]: https://www.contributor-covenant.org
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Comment by joranquinten** [comment]
Processed review, good catches on the typos 🙌

**Review by MichaelDeBoey — DISMISSED** [review]

**Review by MichaelDeBoey — APPROVED** [review]

**Pull request merged**